content/blog/2024-12-03-tmil-11/index.md

```diff
@@ -0,0 +1,227 @@
+* [xilem#749][]: Experimentally increased support for custom view context types.
+
+## Masonry
+
+Masonry is the widget system used by Xilem.
+It provides a non-opinionated retained widget tree, designed as a base layer for high-level GUI frameworks.
+
+* [xilem#670][]: Added some features needed for custom client side decorations, by Marco Melorio.
+* [xilem#735][]: Viktor Strate Kløvedal found and resolved several features which incorrectly private.
+* [xilem#736][]: Added a padding property to `SizedBox`. This PR also exposed this property to Xilem.
+* [xilem#754][], [xilem#755][], [xilem#762][]: Refactored Masonry's text support to reduce internal duplication and validate previously mentioned changes in Parley.
+
+Daniel's talk *Masonry: An Imperative Widget toolkit in Rust* for GOSIM China 2024, was also published [on YouTube](https://youtu.be/99zU4bU4kP8).
+
+<!-- TODO: Do we want to/could we embed the video? -->
+
+## Vello
+
+Vello is our GPU vector renderer.
+It can draw large 2D scenes with high performance, using GPU compute shaders for most of the work.
+
+* [vello#735][], [vello#743][]: Update to wgpu 23.0.1, matching what is used by Bevy 0.15.
+* [vello#722][]: Improving image sampling.
+* [vello#740][]: Correcting support for PNG glyphs in fonts, important for emoji on macOS.
+
+## Parley
+
+Parley is a text layout library.
+It handles text layout, mostly at the level of line breaking and resolving glyph positions.
+In November, the main updates were:
+
+* [parley#158][], [parley#160][]: Worked towards `no_std`. Full `no_std` is currently blocked (only) on the release of a dependency.
+* [parley#154][]: `PlainEditor` was significantly refactored to have a more ergonomic and efficient API.
+* [parley#166][]: AccessKit support was integrated directly into Parley. This was previous possible externally, but the external integration is much simpler.
+* [parley#152][]: Cursor blinking was added to the editor example by [tannal](https://github.com/tannal).
+* [parley#182][]: Font fallback on misconfigured Linux systems was resolved by Richard Hozák.
+* [parley#173][]: Ada Böhm added snapshot testing, which will significantly improve our confidence in future changes.
+* [parley#170][]: Significantly simplified the cursor model, which resolved several longstanding issues in text editing.
+
+Kaur also made several improvements to our CI, in [parley#156][], [parley#171][] and [parley#184][].
+
+## Kurbo
+
+Kurbo has seen some minor updates within the last month and we're planning to publish them in a minor patch release within the next month.
+
+* [kurbo#390][] Reduce number of operations in `Triangle::circumscribed_circle`.
+* [kurbo#399][] Implement `Sum` for `Vec2`.
+
+## Peniko
+
+We've been working towards a big release, which will remove the old `peniko::Color` type and use the new `color` crate instead.
+This release will be the start of a path towards wider color gamut support throughout the Linebender ecosystem.
+## Piet
```

Comment:
Haha, my goal was to get a single empty line here to match the other sections. 😅

```suggestion
This release will be the start of a path towards wider color gamut support throughout the Linebender ecosystem.

## Piet
```